### PR TITLE
Provide a full bundle-update pipeline example

### PR DESCRIPTION
**Pull request by orien**
Add documentation for adding a Buildkite bundle-update pipeline to existing projects. Hopefully this makes it a little easier to get started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,86 @@ steps:
...
 
+## Example Pipeline
+
+This is an example pipeline which ties everything together to produce nicely
+annotated bundle update pull requests.
+
+This pipeline requires two secrets:
+
+* Write access to the project GIT repository, by way of an SSH Key. This write
+  access is used for pushing up the bundle update commit.
```
Comment (patrobinson) [inline README.md]: Can we link to the [deploy key documentation](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) to give people a hint on how to create one?

File: README.md [hunk 2]
```diff
@@ -121,6 +121,86 @@ steps:
...
+
+* Github API access, by populating the environment variable `GITHUB_TOKEN` with
+  a personal access token providing `repo` access to the repository. This is
+  used for opening the pull request and adding comments.
+
+It's recommended to use the [AWS S3 Secrets Buildkite Plugin] to provide these
```
Comment (patrobinson) [inline README.md]: > [AWS S3 Secrets Buildkite Plugin]
Is this mean to reference that plugin?

Comment (orien) [reply README.md]: Yeah, the link is at the bottom of the document. Although when I try it, it links a little deep. 

It now links to the uploading secrets section (7bfbf71).

File: README.md (hunk 1 above)
Comment (orien) [reply README.md]: Yes. Good idea.

File: README.md (hunk 2 above)
Comment (patrobinson) [reply README.md]: Sorry missed that, thanks!

File: README.md (hunk 1 above)
Comment (orien) [reply README.md]: Added in b7c0625.

**Pull request merged**